ci/tasks/bats.sh

```
@@ -123,7 +123,7 @@ EOF
     path: /instance_groups/name=bosh/networks/0/static_ips
   - type: replace
     path: /stemcells/0/version
-    value: $iaas_stemcell_version
+    value: "$iaas_stemcell_version"
```

Shellcheck (0.11.0)

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)